_docs/_latest/configure/notifier/index.md

```diff
@@ -2,4 +2,130 @@
 title: Notifier
 order: 300
 ---
+
 # {{ page.title }}
+
+Notifier can dispatch a variety of messages through various channels
+and varying formats alerting you to events in your environment.
+
+### Notification Types
+
+  * Inventory Summary: Count of the resources added in the latest inventory crawl.
+  * Violation Notifications: Individual violations that have been found from the latest scanner run. 
+  * Cloud SCC Findings: Violations converted to Cloud SCC (Cloud Security Command Center) findings format, that's ingestable by Cloud SCC.
+  
+### Notification Channels
+
+  * Email
+  * Slack
+  * Cloud Storage
+
+### Notification Formats
+
+  * Human-readable data: CSV
+  * Structured data: JSON
+
+## Configuring Notifier
+
+### Inventory Summary
+
+This is a count of what resources have been crawled into inventory,
+and outputted to Cloud Storage bucket.
+
+```yaml
+notifier:
+    inventory:
+      summary:
+        enabled: true
+        data_format: csv
+        gcs_path: gs://path_to_foo_bucket
+```
+
+* `enabled`: `true` or `false`. Enables/disables the notification.
+* `data_format`: `csv` or `json`
+* `gcs_path`: The Cloud Storage bucket to upload the inventory summary.
+
+
+### Violation Notifications
+
+To select how you want Notifier to send violation notifications,
+follow the steps below:
+
+1. Open `forseti-security/configs/forseti_conf.yaml`.
+1. Navigate to the `notifier` > `resources` section.
+
+On a per-resources basis, the following options are available. You can use
+any combination of notifiers for each resource.
+
+* `should_notify`: `true` or `false`.  Whether a violation for each resource
+   should be sent.
+
+* `name`: The name of the notifier you want for each resource. You can specify
+  multiple notifiers for each resource.
+
+  The name must match the actual module name for each notifier in 
+  [forseti-security/google/cloud/forseti/notifier/notifiers](https://github.com/GoogleCloudPlatform/forseti-security/tree/2.0-dev/google/cloud/forseti/notifier/notifiers),
+  such as `email_violations.py`
+
+The notification channels can be any of the following.
+* `email_violations`
+This emails all the violation data via SendGrid. Multiple email recipients are
+delimited by comma, such as: `john@mycompany.com,jane@mycompany.com`.
+
+* `slack_webhook`
+This sends individual violations to a Slack channel via a Slack webhook.
+
+* `gcs_violations`
```

Comment:
In reading this document several times now I think a good format for describing key/value pairs in the yaml are as such.

* `slack_webhook`
  * **Description**: A URL for a pre-configured Slack webhook.
  * **Valid values**: String
  * **Note**: See [#](here) on how to generate a webhook.
* `data_format`
  * **Description**: The format of the data generated for a given violation.
  * **Valid values**: one of `csv` or `json`
  * **Note**: Slack only supports the `json` type